src/components/LandingLabAbout/index.tsx

```diff
@@ -17,19 +16,13 @@ const AboutTitle: FC = ({ children }) => (
 
 const furtherProjects = [
```

Comment:
This variable name doesn't make sense anymore. Let's change it to somethink like `repositories`?

Reply:
I just love these quality reviews. Thank you @dnsos !